Augurpy.ipynb

```diff
@@ -0,0 +1,1453 @@
+{
```

Comment:
<p>Maybe add a sentence at the end like: "Here we are using the original implementation to blabal"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/theislab/pertpy-tutorials/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>